README.md

```diff
@@ -52,49 +53,48 @@ SAMPLE1,/data/Bams/sample1-N.bam,/data/Bams/sample1-T.bam
 This pipeline uses the following tools for generating specific index files.
   - [samtools](https://www.htslib.org/doc/samtools-faidx.html).fai
   - [picard](https://gatk.broadinstitute.org/hc/en-us/articles/360037593331-CreateSequenceDictionary-Picard-).dict 
   - [bwa](https://bio-bwa.sourceforge.net/bwa.shtml).amb, .ann, .bwt, .pac, .sa 
 
+When you run the pipeline, you will use the mandatory `--ref` and `--dict` parameters to specify the location and names of the reference files:
+```
+--ref /path/to/ref.fasta --dict /path/to/ref.dict
+```
 
-***Note***: You must specify the full path for the reference fasta, even if it is in your working directory.
-
-
-### 3. Download files
-#### Download the sub-interval files required for Mutect2 
-  - **Step 1**: Click on the [link](https://console.cloud.google.com/storage/browser/genomics-public-data/resources/broad/hg38/v0;tab=objects?pli=1&prefix=&forceOnObjectsSortingFiltering=false&pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22)))
-  - **Step 2**: Click on [scattered_calling_intervals] and select all checkboxes and click `Download`. 
-  - **Step 3**: To download all the files, you will need to install the utility `gsutil`. Please follow the steps as shown below in a command-line window
-    - Download, unzip and install the excecutable  
-      - curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-457.0.0-linux-x86_64.tar.gz
-      - tar -xf google-cloud-cli-457.0.0-linux-x86_64.tar.gz
-      - ./google-cloud-sdk/install.sh
-      - Close rhe current command-line window and re-open it. The utility `gsutil` will now be available in the path.
-  - **Step 4**:Go back to Step (2) and download the files as suggested.
-
-#### A Panel of Normals (PoN) for Mutect2  
-The user can create a PoN file using our [pipeline](https://github.com/Sydney-Informatics-Hub/Somatic-ShortV?tab=readme-ov-file#4-create-pon-per-genomic-interval).
+**Common biallelic variant resources**  
+Common biallelic variant resources, such as databases that catalog known genetic variants (e.g., gnomAD) can be used to filter out germline variants and avoid false positives. 
+- A bash script `gatk4_selectvariants.pbs` is provided in the `scripts` folder which can read in your public resource VCF, and select common biallelic SNP variants (by default, those with an allele frequency of > 0.05). This script can be directly run on the NCI Gadi HPC. You can fill in the values for the required variables in this script and submit your job using the command :`qsub gatk4_selectvariants.pbs`.
+- We will include this functionality as an optional module in the next version of the pipeline.
```

Comment:
```suggestion
Common biallelic variants databases (e.g. gnomAD) can be used to filter false positive germline variants from your somatic variant dataset. We have provided a script `scripts/gatk4_selectvariants.pbs` for this purpose. It executes[ GATK's SelectVariants](https://gatk.broadinstitute.org/hc/en-us/articles/13832694334235-SelectVariants) tool. This will be included as an optional functionality in this pipeline in an upcoming update.  
```

Reply:
ok